docs/advanced.md

```diff
@@ -0,0 +1,142 @@
+## Advanced usage
+
+In the [quickstart](./quickstart.md), we have created a simple user model and ran a few queries.
+However, Prisma and the Go client are designed to work with relations between models.
+
+We already created a post model, such as for a blog. Let's assume we want to add a user model, and connect these models
+in a way so we can rely on SQL's foreign keys and the Go client's ability to work with relations.
+
+So let's introduce a user model:
+
+```prisma
+model User {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    email     String   @unique
+    name      String?
+    age       Int?
+
+    posts     Post[]
+}
+```
+
+We will also need to add a relation from to the post model in order to make a 1:n relation between those models:
+
+```prisma
+model Post {
+    // ...
+
+    author   User @relation(fields: [authorID], references: [id])
+    authorID String
+}
+```
+
+<details>
+    <summary>Expand to show full schema.prisma</summary>
+
+    ```prisma
+    datasource db {
+        // could be postgresql or mysql
+        provider = "sqlite"
+        url      = "file:dev.db"
+    }
+
+    generator db {
+        provider = "go run github.com/prisma/prisma-client-go"
+    }
+
+    model Post {
+        id        String   @default(cuid()) @id
+        createdAt DateTime @default(now())
+        updatedAt DateTime @updatedAt
+        published Boolean
+        title     String
+        content   String?
+
+        author   User @relation(fields: [authorID], references: [id])
+        authorID String
+    }
+
+    model User {
+        id        String   @default(cuid()) @id
+        createdAt DateTime @default(now())
+        email     String   @unique
+        name      String?
+        age       Int?
+
+        posts     Post[]
+    }
+    ```
+</details>
+
+Whenever you make changes to your model, migrate your database and re-generate your prisma code:
+
+```shell script
+# apply migrations
+go run github.com/prisma/prisma-client-go migrate save --experimental --name "add user model"
```

Comment:
Any way to simplify these long calls?

Reply:
Unfortunately no :( alternatively we could do `prisma-client-go ...` but it won't guarantee the correct version, which would be fine in migrate I think. However I think this could confuse users since we would use different commands so I think this is currently the best way to do it.